storage/storage.go

```diff
@@ -59,6 +59,20 @@ func FindByName(name string) (*Tunnel, error) {
 	return tun, nil
 }
 
+func FindAll() ([]string, error) {
```

Comment:
This method should return a copy of `Store.Tunnels` and the caller function/method should filter what values it is interested on.